jsk_fetch_robot/README.md

```diff
@@ -50,8 +50,42 @@ wget https://raw.githubusercontent.com/jsk-ros-pkg/jsk_roseus/master/setup_upstr
 bash /tmp/setup_upstream.sh -w ../ -p jsk-ros-pkg/geneus -p euslisp/jskeus
 source /opt/ros/$ROS_DISTRO/setup.bash
 rosdep install -y -r --from-paths . --ignore-src
-cd ../
-catkin build fetcheus jsk_fetch_startup
+cd jsk_robot
+git fetch origin
+git checkout -b develop/fetch origin/develop/fetch
+```
+
+Next, you need to install `roseus_resume` package. For more information, please check [Affonso-Gui/roseus_resume](https://github.com/Affonso-Gui/roseus_resume).
+```
+cd ../../  # catkin_ws/src
+# clone euslisp
+wget https://raw.githubusercontent.com/jsk-ros-pkg/jsk_roseus/master/setup_upstream.sh -O /tmp/setup_upstream.sh
+bash /tmp/setup_upstream.sh -w ..
+
+# clone roseus
+wstool set jsk-ros-pkg/jsk_roseus --git https://github.com/jsk-ros-pkg/jsk_roseus.git -v master -u -y
+cd euslisp/Euslisp/
+
+# checkout euslisp/eus-handler
+git remote add Affonso-Gui https://github.com/Affonso-Gui/EusLisp.git
+git fetch Affonso-Gui
+git checkout eus10
+
+# checkout roseus/eus-handler
+cd ../../jsk-ros-pkg/jsk_roseus/
+git remote add Affonso-Gui https://github.com/Affonso-Gui/jsk_roseus.git
+git fetch Affonso-Gui
+git checkout eus10
+cd ../../  # catkin_ws/src
+
+# clone roseus_resume
+git clone https://github.com/Affonso-Gui/roseus_resume.git
+```
+
```

Comment:
I think these things can be merged into `.rosinstall`. Could you update `jsk_fetch_robot/jsk_fetch_user.rosinstall` instead?

Reply:
The above means 
```
# clone roseus
wstool set jsk-ros-pkg/jsk_roseus --git https://github.com/jsk-ros-pkg/jsk_roseus.git -v master -u -y
cd euslisp/Euslisp/
# checkout euslisp/eus-handler
git remote add Affonso-Gui https://github.com/Affonso-Gui/EusLisp.git
git fetch Affonso-Gui
git checkout eus10
# checkout roseus/eus-handler
cd ../../jsk-ros-pkg/jsk_roseus/
git remote add Affonso-Gui https://github.com/Affonso-Gui/jsk_roseus.git
git fetch Affonso-Gui
git checkout eus10
cd ../../  # catkin_ws/src
# clone roseus_resume
git clone https://github.com/Affonso-Gui/roseus_resume.git
```